opds.py

```diff
@@ -1179,9 +1179,27 @@ def license_tags(cls, license_pool, loan, hold):
             return tags
 
 
-        holds_kw = dict(total=str(license_pool.patrons_in_hold_queue or 0))
-        if hold and hold.position:
-            holds_kw['position'] = str(hold.position)
+        holds_kw = dict()
+        total = license_pool.patrons_in_hold_queue or 0
+        position = hold.position or 0
+        if hold:
+            if position:
```

Comment:
This won't work if position is 0.

Reply:
This is by design. [The spec](https://docs.google.com/document/d/1DZCupVm5EnvpIrqRq8ABJi-68vgqwA7CymO9PM7PyPE/edit#heading=h.6ffrgis88ere) shows an example where the patron is in position 0 and the `opds:position` field is missing. I'll change this code to be more explicit.

Reply:
Never mind, I checked that leaving it out for position 0 is the current behavior.